### ISSUE TITLE
year_dendro aanpassen in plotinfo & year bij dendro-files

### ISSUE THREAD

**Issue by leymanan** [inbo/forrescalc#114]
Tot nu hebben we "year of growing season" (variabele `year_dendro` en `year` bij dendrometrie) bepaald door alles na 1 juli bij huidige jaar te rekenen, en alles ervoor tot vorig jaar.
Maar eigenlijk is dat niet helemaal correct, en we zouden dat graag veranderd zien naar
- alles vóór 1 mei bij vorig jaar
- alles na 1 mei bij huidig jaar

Functie `load_plotinfo`: nu 
`year_dendro = year(round_date(.data$date_dendro, "year")) - 1`

Ook aan te passen in functie `load_dendrometry`: nu 
`year = year(round_date(.data$date_dendro, "year")) - 1`

**Comment by ElsLommelen** [comment]
Euh, puur uit interesse: in hoeverre worden er eigenlijk in mei en juni bomen opgemeten?
(Ter herinnering: destijds hebben we die round gebruikt voor de dendro-metingen met het idee om 'alles van de winter bij het voorbije jaar te tellen', en toen maakte het niet veel uit omdat bomen toch enkel in de winter gemeten werden.  Voor metingen in het groeiseizoen, zoals de vegetatie, wordt die round niet gebruikt, zodat hier gewoon het jaar van de meting meegenomen wordt.)

Ik zal het aanpassen voor de dendro-metingen, maar in hoeverre wil je die jaargrens ook gelijk trekken voor de andere metingen (vegetatie,...)?  Wat dan bv. met de voorjaarsflora?

**Comment by leymanan** [comment]
ja, inderdaad, maar blijkbaar toch ook soms metingen in juni ...
En als ik bij documentatie moet zetten dat we groeiseizoen pas laten starten vanaf 1/7, komt dat niet echt professioneel over ...

voor veg en voorjaarsflora is het OK zoals het is!

**Comment by ElsLommelen** [comment]
Nee hoor, zeker geen probleem om het aan te passen.
Tja, als professional kan je je ook afvragen: in hoeverre zijn bomen in juni al gegroeid t.o.v. de winter voordien en is het te motiveren om ze bij het volgende jaar te tellen?  ;-)  Voor wat oudere bomen en metingen om de 10 jaar is er uiteraard geen probleem mee, en maakt een jaartje eerder of later niet zoveel uit, maar voor een experiment met jonge bomen en jaarlijkse metingen zou ik zo'n metingen eerlijk gezegd toch niet zo graag in m'n dataset hebben (afhankelijk van de vraagstelling).

**Comment by leymanan** [comment]
't is dat zowel hoogte- als diktegroei vooral in de periode mei-juni plaatsvindt, dus als er dan toch metingen gebeurd zijn eind juni, toch beter om ze in het nieuwe jaar te plaatsen ...

**Comment by ElsLommelen** [comment]
> 't is dat zowel hoogte- als diktegroei vooral in de periode mei-juni plaatsvindt, dus als er dan toch metingen gebeurd zijn eind juni, toch beter om ze in het nieuwe jaar te plaatsen ...

Ah, ok, weer iets bijgeleerd, bedankt!
(Van hoogtegroei wist ik het onbewust wel, ik wacht altijd tot na Sint-Jan om m'n haag te snoeien, van diktegroei had ik eerlijk gezegd geen idee...)

**Comment by leymanan** [comment]
> Berekening van `year` en `year_dendro` aangepast voor `load_data_dendrometry()` en `load_plotinfo()`: t.e.m. 30 april wordt nog bij het vorige jaar geteld, vanaf 1 mei wordt bij het nieuwe jaar geteld. Of moet 1 mei ook nog bij het vorige jaar?

nee, is goed zo, 1 mei bij huidig jaar

> Ik merk dat ook bij `load_data_deadwood()` dezelfde berekening wordt toegepast. Pas ik het hier ook aan naar de grens van 1 mei, of is het niet de bedoeling dat daar het jaar ook aangepast wordt i.f.v. het groeiseizoen van de bomen?

ik twijfelde even, maar omdat alle output mbt liggend dood hout samen zit met het staande en ook het levende, is het inderdaad een goed idee om dat ook voor `load_data_deadwood()` te doen. Bedankt!

**Comment by ElsLommelen** [comment]
Voila, bij deze is dit issue opgelost, vermoed ik?
